templates/transfers_table.php

```diff
@@ -123,12 +98,13 @@ function clickableHeader($displayName,$trsortcol,$trsort,$nosort) {
         $nextPage  = $offset+$limit;
         $transfersort = Utilities::getGETparam('transfersort','');
         $cgias = Utilities::getGETparam('as','');
-        $nextLink  = "$base&$cgioffset=$nextPage&$cgilimit=$limit&transfersort=$transfersort&as=$cgias$cgiuid$cgiminmax&nextlink=1";
+        $nextLink  = transferHref([$pagerprefix.'offset' => $offset + $limit, $pagerprefix.'limit' => $limit, 'nextlink' => 1]);
+
         
         if( $havePrev ) {
            $prevPage = max(0,$offset-$limit);
-           echo "<td class='pageprev0'><a href='$base&$cgioffset=0&$cgilimit=$limit&transfersort=$transfersort&as=$cgias$cgiuid$cgiminmax'><span class='fa-stack fa-lg'><i class='fa fa-square fa-stack-2x'></i><i class='fa fa-angle-double-left fa-stack-1x fa-inverse'></i></span></a></td>";
-           echo "<td class='pageprev'><a href='$base&$cgioffset=$prevPage&$cgilimit=$limit&transfersort=$transfersort&as=$cgias$cgiuid$cgiminmax'><span class='fa-stack fa-lg'><i class='fa fa-square fa-stack-2x'></i><i class='fa fa-angle-left fa-stack-1x fa-inverse'></i></span></a></td>";
+           echo "<td class='pageprev0'><a href='".transferHref([$pagerprefix.'offset' => 0, $pagerprefix.'limit' => $limit])."'><span class='fa-stack fa-lg'><i class='fa fa-square fa-stack-2x'></i><i class='fa fa-angle-double-left fa-stack-1x fa-inverse'></i></span></a></td>";
+           echo "<td class='pageprev'><a href='".transferHref([$pagerprefix.'offset' => $prevPage + $limit, $pagerprefix.'limit' => $limit])."'><span class='fa-stack fa-lg'><i class='fa fa-square fa-stack-2x'></i><i class='fa fa-angle-left fa-stack-1x fa-inverse'></i></span></a></td>";
```

Comment:
Adding $limit makes the prev button ineffective...

`$pagerprefix.'offset' => $prevPage + $limit`
=>
`$pagerprefix.'offset' => $prevPage`